tests/integration/test_fim/test_registry/test_registry_report_changes/test_registry_file_size_values.py

```diff
@@ -70,28 +73,26 @@
 
 # Variables
 
-key = "HKEY_LOCAL_MACHINE"
-sub_key_1 = "SOFTWARE\\test_key"
-sub_key_2 = "SOFTWARE\\Classes\\test_key"
+key = WINDOWS_HKEY_LOCAL_MACHINE
+sub_key_1 = MONITORED_KEY
+sub_key_2 = MONITORED_KEY_2
 
 test_regs = [os.path.join(key, sub_key_1),
              os.path.join(key, sub_key_2)]
-
+reg1, reg2 = test_regs
 test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
 wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
-reg1, reg2 = test_regs
-size_limit_configured = 10 * 1024
+size_limit_configured = SIZE_LIMIT_CONFIGURED_VALUE
```

Comment:
Same as before

Reply:
Use directly  SIZE_LIMIT_CONFIGURED_VALUE instead of defining a new variable size_limit_configured